Add option to show power stats

cjw85 opened the pull request:
Adds ability to show optionally power usage and limit (grouped with temperature and utilisation).

wookayin commented on gpustat.py:
```diff
@@ -114,6 +114,24 @@ def utilization(self):
+        Returns the GPU power usage in Watts,
+        or None if the information is not available.
+        """
+        v = self.entry['power.use']
```
Actually I think it should be "power.draw". A convention here is that I am using the same column name as shown in `nvidia-smi --help-query-gpu`.

cjw85 commented:
Changed: `power.use` -> `power.draw`, and `power.limit` -> `enforced.power.limit` as given by `nvidia-smi --query`. Changed API properties to be consistent (whilst not being verbose for the second case).

wookayin commented:
Between `power.limit` and `enforced.power.limit`, what is the quantity you are seeking to print out? I thought it was `power.limit`, but you do `enforced.power.limit`, right?

cjw85 commented:
I'm calling `N.nvmlDeviceGetEnforcedPowerLimit` so `enforced.power.limit` is the correct form. This is described as "the minimum of various power limiters.", so seems to be the most relevant to display in the simple interface (and provide in the simple API).

wookayin commented:
Merged, thanks!
